todo_app/templates/macros.html

```diff
@@ -0,0 +1,61 @@
+{%  macro item_component(item) %}
+    <li class="list-group-item">
+        <div style="position:relative;right:0;bottom:0">
```

Comment:
would recommend using the bootstrap classes for this if possible, or perhaps a css file. inline styling isnt the nicest to read & doesn't self document very well